.asf.yaml

```diff
@@ -31,7 +31,14 @@ github:
     squash: true
     merge: false
     rebase: false
-  protected_branches: ~
+  protected_branches:
+    main:
+      required_pull_request_reviews:
+        dismiss_stale_reviews: true
+        required_approving_review_count: 1
+      required_signatures: true
```

Comment:
may I ask why signatures is needed? seems developer need re-import ssh key with signed key in Github